addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -882,7 +882,32 @@ export class WysiwygAdapterComponent extends Wysiwyg {
         }
         this.__savedCovers[resModel].push(resID);
 
-        var cssBgImage = $(el.querySelector('.o_record_cover_image')).css('background-image');
+        const imageEl = el.querySelector('.o_record_cover_image');
+        let cssBgImage = imageEl.style.backgroundImage;
+        if (imageEl.classList.contains("o_b64_image_to_save")) {
+            imageEl.classList.remove("o_b64_image_to_save");
+            const groups = cssBgImage.match(/url\("data:(?<mimetype>.*);base64,(?<imageData>.*)"\)/)?.groups;
+            if (!groups.imageData) {
+                // Checks if the image is in base64 format for RPC call. Relying
+                // only on the presence of the class "o_b64_image_to_save" is not
+                // robust enough.
+                return;
+            }
```

Comment:
@bso-odoo maybe you should allow normal `_saveCoverProperties()` behaviour instead of breaking the flow :thinking:, since the `"o_b64_image_to_save"`check is not robust enough e.g.:

https://github.com/odoo/odoo/assets/25032876/dce9243b-780a-459a-9be9-e6819374afda

Here I just set first a `.JPEG` to add the `o_b64_image_to_save` class on the DOM, and replace it immediately by a `.GIF`. The code should save the cover in a normal way and ignore the `WEBP` process, but because of the remaining `o_b64_image_to_save` class, the first save will always be blocked... What do you think?

Reply:
I instead preferred to remove the class when the image is changed, to avoid that the DOM "lies" about the situation.